.gitlab/source_test/notify.yml

```diff
@@ -8,10 +8,9 @@ unit_tests_notify:
     - when: always
   script:
     - source /root/.bashrc
-    - export GITHUB_KEY_B64=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.platform-github-app-key)
-    - export GITHUB_APP_ID=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.platform-github-app-id)
-    - export GITHUB_INSTALLATION_ID=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.platform-github-app-installation-id)
+    - set +x
```

Comment:
Confirming this `set +x` _should_ be added, I've learned to be cautious 😉

Reply:
`set +x` is disabling printing so it's a double precaution. It's probably unnecessary here though